compiler/rustc_codegen_llvm/src/asm.rs

```diff
@@ -472,6 +476,13 @@ pub(crate) fn inline_asm_call(
         let constraints_ok = llvm::LLVMRustInlineAsmVerify(fty, cons.as_ptr().cast(), cons.len());
         debug!("constraint verification result: {:?}", constraints_ok);
         if constraints_ok {
+            if unwind && llvm_util::get_version() < (13, 0, 0) {
+                bx.cx.sess().span_fatal(
+                    line_spans[0],
+                    "the llvm backend doesn't support unwinding from inline assembly.",
```

Comment:
Can you please indicate that this is a limitation of the specific LLVM version against which rustc is compiled?

Reply:
`unwinding from inline assembly is only supported on llvm >= 13`

does this sound ok?

Reply:
I think so.